Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -150,8 +152,16 @@ protected void Page_Load(object sender, EventArgs e)
                 //Skin js requried because using for openpopup update memeber Profile when user is registered user 
                 //VjDefaultPath used in Skin.js for loading icon.
                 WebForms.RegisterClientScriptBlock(Page, "DefaultPath", "var VjThemePath='" + Page.ResolveUrl("~/Portals/_default/vThemes/" + Core.Managers.ThemeManager.CurrentTheme.Name) + "'; var VjDefaultPath='" + Page.ResolveUrl("~/DesktopModules/Vanjaro/UXManager/Library/Resources/Images/") + "'; var VjSitePath='" + Page.ResolveUrl("~/DesktopModules/Vanjaro/") + "';", true);
+
+
+
+
+
+
+
+
                 ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/Portals/_default/Skins/Vanjaro/Resources/js/skin.js"), 2, "DnnFormBottomProvider");
-                ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/DesktopModules/Vanjaro/Common/Frameworks/Bootstrap/5.0.0/js/bootstrap.min.js"), 1, "DnnFormBottomProvider");
+                ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/DesktopModules/Vanjaro/Common/Frameworks/Bootstrap/4.5.0/js/bootstrap.min.js"), 1, "DnnFormBottomProvider");
```

Comment:
Please undo the line for bootstrap change the version 5.0.0 to 4.5.0

Reply:
Merging upstream develop branch has resolved this